src/component/footer/proxy-status.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const {string, oneOf, func} = React.PropTypes;
+
+const ProxyStatus = ({proxyStatus, proxyMessage, proxyWindow}) => {
+  const icon = proxyStatus !== 'offline' ? 'fa fa-circle' : 'fa fa-circle-o';
+  const label = 'Proxy: ' + (proxyMessage ? proxyMessage : proxyStatus);
+  const classes = ['proxy-status'];
+
+  if (proxyWindow) {
+    classes.push('with-info');
+  }
+  classes.push(proxyStatus);
```

Comment:
Why not just `const classes = ['proxy-status', proxyStatus]` above, while you're at it?